scrypto/src/resource/system.rs

```diff
@@ -0,0 +1,99 @@
+use crate::engine::{api::*, call_engine};
+use crate::resource::*;
+use crate::rust::collections::HashMap;
+use crate::rust::string::String;
+
+/// Represents the Radix Engine resource subsystem.
```

Comment:
Rather than subsystem perhaps call it a driver?

Reply:
We may call it "driver" (as loadable module) in Radix Engine. However, this is a developer facing API; driver is less appropriate IMO.